develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown

```diff
@@ -7,14 +7,8 @@ filters and sorting order can be included in the JSP, but that can crowd the JSP
 pretty quick. We recommend instead that you move this functionality to a 
 separate java class, which we refer to as a display context throughout this 
 tutorial. A Display Context is a Java class that defines the configuration for 
-an app's UI elements. 
-
-Creating filters and sorting results involves the following process: 
-
-- Creating the Filters
-- Creating a Comparator class for each field you want to sort
-- Including the selected column and sorting order in the portlet URL
-- Including the selected column and sorting order in the search container
+an app's UI elements. This tutorial explains how to create this filters in a 
```

Comment:
@jbalsas @carloslancha I've simplified the tutorial down to what's in this commit. when you say use an existing comparator, where can I find those so I can list them in the tutorial for others to use? Or do you think these steps are all that is required? Thanks!

Reply:
Cleared this up with Chema. Thanks!